### DISCUSSION TITLE
تولید فیلترشکن رایگان برای دانشجویان

### DISCUSSION THREAD

**Discussion by hiddify-com** [WeAreMahsaAmini/Core#290] [Ideas]
برخی سرویس دهنده های ابری مثل مایکروسافت آژور به دانشجویان همه رشته ها کردیت رایگان ارائه میده که میتونند ازش هر استفاده ای کنند
با استفاده از این کردیت به راحتی میشه دو vps راه اندازی کرد که پروکسی تلگرام و پروکسی وبسایت ها را داشته باشه برای یکسال که هر سال هم میشه رایگان تمدید کرد  (اگر دانشجو بمانید)
توی سایت توضیحات کامل را ارائه کردیم و اسکریپت و آموزش تهیه شده
https://hiddify.github.io/setup_proxy.html

تصور کنید چقدر دانشجو خارج از کشور داریم و هرکدوم چقدر دوست دانشجو دارن که اونا هم میتونن کمک کنند
به راحتی کلی خانواده را میتونه پوشش بده
فقط حجم ماهیانه 100 گیگ رایگان داره که برای خیلی از خانواده ها کافی هست

اگه به هر خانواده ای از این طریق پروکسی برسونیم کلی خانواده میتونن از اینترنت آزاد استفاده کنند

ضمنا این اسکریپت فقط سایت های خارجی را از فیلترشکن رد میکنه و در نتیجه خیلی بار سرور میاد پایین و همینطور حجم نیم بها در سایت های داخلی سرجاش میمونه

داریم روش کار میکنیم که فقط سایت های فیلتر را از فیلترشکن عبور دهیم.

حمایت کنید

**Comment by hiddify-com** [comment]
این موضوع خیلی مهمه چرا کسی حمایت نمیکنه؟

**Comment by AnonymousMaximus2** [comment]
من هستم ، چه کمکی میتونم بکنم؟

من میتونم توی کارهای زیر بهتون کمک کنم

1- ثبت و بالا آوردن سایت اینترنتی مستقل برای این پروژه

2- برنامه نویسی اپلیکیشن موبایل  اندروید و گذاشتن توی گوگل پلی 

3- ایجاد فروشگاه اینترنتی متصل به پی پل جهت دریافت کممکهای مردمی برای توسعه پروژه شما

4- ثبت شرکت در اروپا یا آمریکا یا کانادا یا یک کشور بیطرف که بتونه این پروژه را حمایت و تجاری سازی کنه و هزینه های پروژه را تامین کنه

5- تهیه و تولید محتوی آموزشی فیلم و کلیپ و گذاشتن اونها توی یوتیوب، اینستاگرام، فیسبوک، لینکدین و ...

6- تبلیغات و بااریابی و مارکتینگ

7- کمک به برنامه نویسی و تولید اپلیکیشن تحت ویندوز 

8- مانتایز کردن اپلیکیشنها و تبلیغات و تامین مخارج پروژه چون بعد از مدتی ممکنه افراد زیادی از این سیستم استفاده کنند و سرورهای رایگان آزور جواب نده و مجبور بشیم سرور بخریم

کم کم که اینکارها را انجام بدیم بعدها میتونیم برنامه نویسی گوشی های اپل ایفون و سایر پلتفرمها را هم کمک بگیریم

کارتون خوبه ادامه بدید و من هم برنامه ریزی میکنم هر جا بتونم کمکتون بکنم

**Reply by hiddify-com** [reply]
خیلی عالی ممنون از حمایتتون
1-من سایت را درست کردم
2-هیچ اپلیکیشن خاصی نمیخواد، همش به صورت رایگان هست توی اینترنت فقط کانفیگ هست که اون هم با یک لینک ساده آماده کردم
3-بحث حمایت مالی که خوب خیلی موثره در رشد یه پروژه 
5 و 6- این خیلی مهم هست و مورد نیاز توی این زمینه خیلی کمک نیاز دارم
8- بحث استفاده از آژور اینه که رایگان باشه و اینجوری بتونیم کلی دانشجو را با دانش صفر برنامه نویسی و کامپیوتر همراه خودمون کنیم ولی در حالت کلی آژور گرونه و برای کاربرد ما نمیصرفه و 
من آمادگی اینو دارم که تو دیتاسنتر های مختلف اینو راه اندازی کنم برای عموم مردم. من هدفم اینه که جلوی فیلترینگ فله ای رو هم بگیرم تا فیلترچی نتونه فله ای ما رو فیلتر کنه

اگه موضوع بازاریابی حل بشه و آدما ببینن و ویدیو آموزشی ساخته بشه 
من میتونم تمرکز بیشتری روی آماده کردن و قوی تر کردن پکیج بذارم

ضمنا این کار روی همه پلتفورم های اندروید، ویندوز، مک آماده و قوی هست فقط برای iOS نیاز هست که اپلیکیشنش سایدلود بشه

**Comment by AnonymousMaximus2** [comment]
شما داخل ایران هستید یا خارج از ایران؟

اگر در آمریکا هستید من میتونم این پروژه شما را تبدیل به استارتاپ کنم و براتون جذب سرمایه کمک کنم که بتونید سرمایه جذب کنید و با سرمایه ای که سرمایه گذارها میدن سرورهای زیادی را بخرید و خدمات رایگان بیشتر و با کیفیت بیشتری بدهید،

من هم توی تیم شما کمک میکنم.

**Reply by hiddify-com** [reply]
اگر سرمایه گذار باشه لطفا به من ایمیل بزنید hiddify@gmail.com

**Reply by AnonymousMaximus2** [reply]
سرمایه گذارهای با تجربه که سرمایه زیادی دارند بصورت خام سرمایه گذاری نمیکنند

با مراحلی که من توضیح دادم وارد سرمایه گذاری میشوند.

یعنی اول شخص را برررسی میکنند تا اعتماد کنند

دوم  تیمش را بررسی میکنند که تیم قوی و حرفه ای باشد

سوم بخشی از پروژه را انجام داده باشد و پروژه به اصطلاح ران Run شده باشد.

چهارم سیستم شفاف مالی باشد که مطمئن باشند سرمایه ای که میگذارند درست مصرف میشود که برای این کار بهترین راه ثبت یک شرکت است.

**Reply by AnonymousMaximus2** [reply]
اگر بتوانید یک تیم چند نفره در آمریکا جمع کنید بقیه کارها را من برای جذب سرمایه پیگیری میکنم. چون در آمریکا جذب سرمایه اسانتر از سایر کشورها است.

البته کشورهای اروپائی مانند آلمان و انگلیس و حتی کشورهای عربی مانند امارات هم میشه سرمایه گذار پیدا کرد.

ولی آمریکا یک سری قوانین و مقررات و پتانسیل ها داره که باعث میشه کار جذب سرمایه آسانتر پیش بره

**Comment by AnonymousMaximus2** [comment]
میشه کلیه این تنظیمات را بصورت یک داکر درآورد و توی داکر هاب رجیستر کرد که فقط با نصب داکر بشه اجراء کرد و دیگه نیاز به این همه کار و تنظیمات نباشه ؟

مثلا داخل داکر یک وب سرور مثل تنظیمات مودمهای ای دی اس ال بزاریم که هر کسی داکر را نصب کرد با وب سرور بتونه داخل داکر بره  و تنظیمات خودش را بزاره و داکر را ریستارت کنه ؟

**Reply by hiddify-com** [reply]
این مراحل برای راه اندازی یک vps هست 
کانفیگش اتوماتیک انجام میشه
ابتدا داکر استفاده شده بود ولی برای بالابردن کارایی روی vps های ضعیف مستقیم نصب میشه و اتومات هست

**Comment by AnonymousMaximus2** [comment]
اگه سایت و هاستینگ داری یک ورد پرس داخلش نصب کن و فروشگاه فایل را با ووکامرس یا Easy Digital Downloads راه اندازی کن

فعلا اپلیکیشن های موبایل اندروید و اپل و ویندو و مک را رایگان بزار برای داونلود 

ولی باید محصولی که امکانات بیشتر دارد را تولید کنی و برای فروش بگذاری

درگاه پرداخت پی پل برایش درست کن تا هر کسی خواست بتونه پول کمک پرداخت کنه

اگر توی آمریکا هستی با چند صد دلار میتونی یک شرکت استارتاپ کلا آنلاین ثبت کنی 

بعد کم کم راهنمائی میکنم کمپین جذب سرمایه برای خرید چند سرور قوی راه اندای کنی

توی دیتا سنترهای اروپایی سرورهای خوب Dediated ماهیانه با 40 دلار میشه اجاره کرد که داخلش میشه دهها سرور Virtual Machine بالا آورد.

**Reply by hiddify-com** [reply]
محدودیت روی VPS نیست مشکل IP هست و هر آی پی باید از رنج های مختلف باشه تا فله ای فیلتر نشه
ایده من فعلا روی به کار گیری دانشجوهاست و بدون هزینه هست
ولی برای اون موضوع نیاز به سرمایه گذار دارم که بعید میدونم به راحتی باشه

**Reply by AnonymousMaximus2** [reply]
سرمایه گذار ها ادبیات و روش خاصی دارند که در بالا شرایطش را توضیح دادم

یه کمی سرمایه جذب کنید میتوانید یک رنج آی پی را بخرید ، هزینه زیادی ندارند،  IP6  از IP4 فکر میکنم ارزانتر باشد.

**Reply by hiddify-com** [reply]
ایران IPv6 را ساپورت نمیکنه و در نتیجه  قابل استفاده نیست
مشکل رنج IP هم اینه که راحت یکی از آی پی ها شناسایی بشه کل رنجو بلاک میکنند
پس بهترین کار اینه که ipv4 تکی بگیریم

بحث وقتی سراغ فیلترشکن میشه آدم باید کاملا مخفی باشه وگرنه شناسایی میشه و دردسر میشه برای اون ادم
و سرمایه گذار های عادی دنبال سود مالی هستند از این پروژه ولی ما هدفمون سود رسوندن به مردمه که در نتیجه سرمایه گذار متفاوتی نیاز داره

**Reply by norsnap** [reply]
> محدودیت روی VPS نیست مشکل IP هست و هر آی پی باید از رنج های مختلف باشه تا فله ای فیلتر نشه ایده من فعلا روی به کار گیری دانشجوهاست و بدون هزینه هست ولی برای اون موضوع نیاز به سرمایه گذار دارم که بعید میدونم به راحتی باشه

اتفاقا الان محدودیت IP زیاد مشکلی نیست مشکل پروتوکل هستش وقتی VPN میخواد وصل بشه به اصطلاح hand shake میکنه و موقع هند شیک میتونن متوجه بشن که این اتصال VPN هستش و نمیزارن اتصال صورت بگیره

**Reply by norsnap** [reply]
حتی پروتوکل mtproto تلگرام هم موقع هند شیک به مشکل میخوره و نمیتونه وصل بشه. پروتوکل پروکسی برنامه سیگنال گویا این مشکل رو حل کرده و شبیه هند شیک https وبسایت هستش.
پروتوکل shadowsocks+v2ray+websocket+tls+cdn هم وقتی از بالا بهش نگاه کنی انگار کاربر داره یک وبسایت رو بازدید میکنه و هیچ چیز مشکوکی نمیبینن که کانکشن رو ببندن.
اینجا روش راه اندازیش رو توضیح دادم
https://github.com/WeAreMahsaAmini/Core/discussions/62

**Reply by hiddify-com** [reply]
MTProxy TLS
مشکل هندشیک رو حل کرده 
فیلترچی فکر میکنه سایت گوگل هست و اگر فیلترچی بخواد تست کنه ببینه گوگل هست یا نه دقیقا سرتیفیکیت تولید شده گوگل را پس می فرسته و قابل شناسایی نیست

**Reply by hiddify-com** [reply]
پروکسی های TLS خیلی سخت قابل شناسایی هستن
این قضیه cdn موقتی هست و قطعا به زودی بسته میشه

**Reply by hiddify-com** [reply]
ضمن اینکه خود استفاده از websocket مشکوک میکنه ترافیک رو

**Reply by andishehk** [reply]
بچه ها به نظرم این پروژه رو توی ایده[ وی پی ان](https://github.com/WeAreMahsaAmini/Core/discussions/249) ادغام کنین. آرش تونسته سرور وی پی ان خیلی ارزون بیاره بالا و همه میتونن از اون سیستم استفاده کنن و یه سرور برای خودوشن و دوستا و خانواده هاشون بیارن بالا. کامنتشو [اینجا](https://github.com/WeAreMahsaAmini/Core/discussions/249#discussioncomment-3772450) چک کنین و باهاش تماس بگیرین اگه سوالی دارین.

**Reply by AnonymousMaximus2** [reply]
بله دیتا سنتر Hetzner خیلی ارزونه

**Reply by hiddify-com** [reply]
بله ارزونه
ولی هتزنر آی پیش گرونه و پروکسی هایی که پخش میشن سریع فیلتر میشه
ولی پروکسی اختصاصی دیگه فیلتر نمیشه چون خیلی کم استفاده میکنند
ضمن اینکه ایده من رایگانه و برای دانشجوها هست 
فقط با یک کلیک میتونن یه فیلترشکن خوب درست کنند
لینک رو ببین
https://hiddify.github.io/setup_proxy.html

**Comment by EhsanKey** [comment]
سلام و درود 
شماره ایران نداره برای ثبت نام 
چ کنیم؟

**Reply by Dorood-bar-shoma** [reply]
درود ،ببخشید،چون تخصصی در این علم ندارم سوالی دارم که شاید برای شما خیلی بی مورد و مسخره باشه.
چگونه می توانم بفهمم که لینکی که شما یا هر شخص دیگه ایی می‌گذارد و میگن روی اون کلیک کنید متعلق به یک حکر نباشه و قصدش ورود به حسابها و فایلهای شخصیم نباشه؟
سپاسگذاریم
